trailblazer/store/models.py

```diff
@@ -91,7 +91,8 @@ class Analysis(Model):
     uploaded_at = Column(types.DateTime)
     workflow_manager = Column(types.Enum(*WorkflowManager.list()), default=WorkflowManager.SLURM)
 
-    jobs = orm.relationship("Job", cascade="all,delete", backref="analysis")
+    jobs = orm.relationship("Job", cascade="all,delete", foreign_keys="Job.analysis_id")
```

Comment:
Just checking 
`cascade="all,delete"` will transfer all changes made in the analysis table over to jobs if applicable. 
`cascde="ondelete"` will delete rows in the jobs when an analysis disappears for which the job is the foreign key. 

Is the bahviour of "all,delete" wanted here or is the "ondelete" enough? Maybe it doesnt matter

Reply:
I think you are right that delete is enough but I'll leave it as it was